client/scss/components/_body_content.scss

```diff
@@ -91,6 +91,7 @@ $center-width-xlarge: (6/6) * 100%;
   left: 50%;
   margin-left: -50vw;
   padding: 0 map-get($container-padding, 'small');
+  background: color('cream');
```

Comment:
I have vague memories of removing this as it had been causing problems elsewhere – maybe the transporters at the bottom of the digital stories?